src/PharoThings-Devices-HCSR04.package/PotHCSR04Device.class/instance/rebootSensor.st

```diff
@@ -0,0 +1,7 @@
+commands
+rebootSensor
+	"reset the pins when the sensor freeze"
+	echoPin beDigitalOutput; value: 1.
+	1 milliSeconds wait.
+	echoPin value: 0; beDigitalInput; enablePullDownResister.
+	triggerPin beDigitalOutput; value: 0
```

Comment:
and triggerPin is always in output mode. So it can be also extracted to #connect

Reply:
It is always output mode when are 4 PINs. 
In the method configureForRpiBoard3B if the echoPin is Nil it receives the triggerPin (when is set using the method signalPin, this method is to use when has only 3 PINs, I wrote in code comments). 
This way, it controls the same PIN using different behaviors.